lib/activity/router.js

```diff
@@ -20,8 +20,10 @@ module.exports = function route(callback) {
 
       const promise = Promise.all(subscriptions.map(async (subscription) => {
         if (!subscription.isEnabledForGitHubEvent(context.event)) {
+          context.log(`Checking subscription for event type: ${context.event}. (webhooks.subscription.match=0)`);
```

Comment:
We should double check whether we need to set an explicit log level here so that these will show up in Loggly. I don't remember exactly. 🤔

Reply:
req.log (without passing the level) is in loggly:

https://github.com/integrations/slack/blob/e2ba204afb80619918929dd42804174374c0e579/lib/api.js#L51

`context.log.debug` (debug log level) is not.
https://github.com/integrations/slack/blob/e2ba204afb80619918929dd42804174374c0e579/lib/activity/router.js#L19

Reply:
Awesome! 

While we're here, let's also remove the `.debug` from line 19? I feel like we do definitely want to log that.